internal/k8shandler/collection.go

```diff
@@ -593,6 +599,29 @@ func (clusterRequest *ClusterLoggingRequest) createOrUpdateCollectorServiceAccou
 	}
 }
 
+func (clusterRequest *ClusterLoggingRequest) addSecurityLabelsToNamespace() error {
+	ns := &corev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: clusterRequest.Cluster.Namespace,
+		},
+	}
+
+	if err := clusterRequest.Get(ns.Name, ns); err == nil {
```

Comment:
better pattern is to check `err != nil` , write log and return. then continue with adding labels